app/sbna-poc/v1_0__initial-form/fields.ts

```diff
@@ -10,6 +10,36 @@ const immigrationAccommodationHint = {
   `,
 }
 
+const impactOfDrugUsePhysicalHint = {
```

Comment:
We could specify these directly on the field, for example

```javascript
{
  some_field: {
    // other properties..
    hint: { text: 'the hint text' }
  },
}
```

Reply:
Done